webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const HtmlPlugin = require('html-webpack-plugin');
```

Comment:
Another way to stop eslint from yelling is to add this line to the top of your webpack:
```suggestion
/* eslint-env node */
const HtmlPlugin = require('html-webpack-plugin');
```